x-pack/plugins/observability/server/routes/slo/route.ts

```diff
@@ -114,6 +114,42 @@ const createSLORoute = createObservabilityServerRoute({
   },
 });
 
+const inspectSLORoute = createObservabilityServerRoute({
+  endpoint: 'POST /internal/api/observability/slos/_inspect 2023-10-31',
```

Comment:
@shahzad31 Out of curiosity, I have seen this `2023-10-31` in the other routes as well and I wanted to ask what is it about (my guess is api versioning?) and how do we know which date to add?

Reply:
Yes, it's a kibana API version, for now that's the only version allowed